go-common.mk

```diff
@@ -164,10 +164,11 @@ endif # GORUNGET
 
 $(GOTESTCOVERHTML): $(GOTESTCOVERRAW)
 	$(GO) tool cover -html=$< -o $@
-	# To allow Code Climate to understand our uploaded coverage files
-	case "$$(grep '^module' go.mod | awk -F/ '{print$$NF}')" in \
+	@# To allow Code Climate to understand our uploaded coverage files
+	@case "$$(grep '^module' go.mod | awk -F/ '{print$$NF}')" in \
```

Comment:
So @ == quiet and presumably @case will apply to all the commands within the case structure. What's the point of putting @ on the comment line?

Reply:
Otherwise the comment line gets echoed during the build